.sass-lint.yml

```diff
@@ -4,6 +4,7 @@ files:
   ignore:
     - 'client/src/styles/legacy/*'
     - 'admin/client/src/styles/legacy/*'
+    - 'src/Dev/**/*'
```

Comment:
Shouldn't it just ignore all of `src/`, since we don't expect CSS or JS files there?